internal/pkg/views/demomode.go

```diff
@@ -20,13 +45,41 @@ func stripSecretVals(s string) string {
 	fqdnRegex := regexp.MustCompile(`"fqdn":\s*".+?(?:\\"|[^"])*"`)
 	s = fqdnRegex.ReplaceAllString(s, `"fqdn": "HIDDEN-FQDN"`)
 
-	nameRegex := regexp.MustCompile(`"name":\s*".+?(?:\\"|[^"])*"`)
-	s = nameRegex.ReplaceAllString(s, `"name": "HIDDEN-NAME"`)
+	scmURIRegex := regexp.MustCompile(`"scmUri":\s*".+?(?:\\"|[^"])*"`)
+	s = scmURIRegex.ReplaceAllString(s, `"scmUri": "HIDDEN-URI"`)
 
 	storageKeyRegex := regexp.MustCompile(`([A-Za-z/+0-9]{86})(==)`)
 	s = storageKeyRegex.ReplaceAllString(s, "HIDDEN-KEY")
 
+	// this jwt regex only matches if the full string contains only a token (e.g. ^...$)
+	jwtTokenRegex := regexp.MustCompile(`^(["]?)[A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*(["]?)$`)
+	s = jwtTokenRegex.ReplaceAllString(s, "${1}HIDDEN-JWT${2}")
+
+	anyKeyRegex := regexp.MustCompile(`(".*[kK]ey":\s*").+?(?:\\"|[^"])*(")`)
+	s = anyKeyRegex.ReplaceAllString(s, "${1}HIDDEN-KEY${2}")
+
+	anySecretRegex := regexp.MustCompile(`(".*[sS]ecret":\s*").+?(?:\\"|[^"])*(")`)
+	s = anySecretRegex.ReplaceAllString(s, "${1}HIDDEN-SECRET${2}")
+
+	anyPasswordRegex := regexp.MustCompile(`(".*[pP]assword":\s*").+?(?:\\"|[^"])*(")`)
+	s = anyPasswordRegex.ReplaceAllString(s, "${1}HIDDEN-PASSWORD${2}")
+
 	sshRegex := regexp.MustCompile(`ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3} ([^@]+@[^@]+)`)
```

Comment:
```suggestion
	sshRegex := regexp.MustCompile(`ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3} ([^@]+@[^@]+)?`)
```

This isn't part of the PR - noticed while testing that an SSH key I have didn't get stripped as it didn't have an email address.

Reply:
Probably better to track in a separate issue and I can add a test too.

Reply:
I think i'll just do this here if that's okay (incl test) that way it's in the same version bump. I'll proceed as if you've said "yes Ben that's great please do that" unless I hear otherwise :)